test/nodes/test_table_reader.py

```diff
@@ -9,45 +9,43 @@
 
 
 @pytest.fixture
-def table1():
+def table_doc1():
     data = {
         "actors": ["brad pitt", "leonardo di caprio", "george clooney"],
         "age": [58, 47, 60],
         "number of movies": [87, 53, 69],
         "date of birth": ["18 december 1963", "11 november 1974", "6 may 1961"],
     }
-    return pd.DataFrame(data)
+    return Document(content=pd.DataFrame(data), content_type="table", id="doc1")
 
 
 @pytest.fixture
-def table2():
+def table_doc2():
     data = {
         "actors": ["chris pratt", "gal gadot", "oprah winfrey"],
         "age": [45, 36, 65],
         "number of movies": [49, 34, 5],
         "date of birth": ["12 january 1975", "5 april 1980", "15 september 1960"],
     }
-    return pd.DataFrame(data)
+    return Document(content=pd.DataFrame(data), content_type="table", id="doc2")
 
 
 @pytest.fixture
-def table3():
+def table_doc3():
     data = {
         "Mountain": ["Mount Everest", "K2", "Kangchenjunga", "Lhotse", "Makalu"],
         "Height": ["8848m", "8,611 m", "8 586m", "8 516 m", "8,485m"],
     }
-    return pd.DataFrame(data)
+    return Document(content=pd.DataFrame(data), content_type="table", id="doc3")
 
 
+@pytest.mark.integration
```

Comment:
I added the `@pytest.mark.integration` labels to the appropriate tests (all of them) since they all load an external model from HF.